v2/internal/frontend/desktop/windows/go-webview2/pkg/edge/ICoreWebView2Controller.go

```diff
@@ -1,5 +1,6 @@
 package edge
 
+import "C"
```

Comment:
This is not required and will force the user to have gcc installed.

Reply:
removed